src/services/wallets/torus.ts

```diff
@@ -47,9 +47,10 @@ export const getPublicAddress = async (
 };
 
 export const getWalletAddress = async (): Promise<string> => {
-  const { verifier, verifierId } = await getUserInfo();
+  //const { verifier, verifierId } = await getUserInfo();
+  const { verifierId } = await getUserInfo();
   const walletAddressHolder = await getPublicAddress(
-    verifier as VerifierTypes,
+    "google" as VerifierTypes,
```

Comment:
I'm guessing this is waiting for #18 ?

Reply:
Yes.  there is a bug here and this is a work around